README.md

```diff
@@ -140,17 +140,6 @@ See the `fi_gni(7)` man page for more details.
 
 - The `gni` provider requires `gcc` version 4.9 or higher.
 
-### mxm
-
-***
-
-The `mxm` provider runs over the MXM (Mellanox messaging) interface that is
-currently supported by the Mellanox InfiniBand fabrics.  The `mxm` provider
-makes use of MXM tag matching API in order to implement a limited set of the
-Libfabric data transfer APIs, namely, tagged message queue.
-
-See the `fi_mxm(7)` man page for more details.
-
```

Comment:
Please keep an mxm section, but state that the provider was deprecated and removed after the 1.4 release.  You can remove the other text.